apps/backend/test/factories/adoption.ts

```diff
@@ -0,0 +1,20 @@
+import { Factory } from 'fishery';
+import { faker } from '@faker-js/faker';
```

Comment:
Please include these dependencies in the app's package.json.

Reply:
Thank you for your feedback, I have addressed it now (hopefully)